products/zoningtaxlots/models/product/qaqc/qc_bbldiffs.sql

```diff
@@ -16,6 +16,7 @@ bbldiffs AS (
         a.tax_block,
         a.tax_lot,
         a.bbl AS bblnew,
+        a.geom,
```

Comment:
Let's put this column last in the select statement - just a bit of a norm we have